quantum/nodes/not-gate/not-gate.js

```diff
@@ -0,0 +1,36 @@
+module.exports = function(RED) {
+  'use strict';
+
+  function NodeTemplateNode(config) {
+    RED.nodes.createNode(this, config);
+    const node = this;
+  }
+
+  function NotGate(config){
```

Comment:
```suggestion
  function NotGateNode(config){
```
The node function should end with "Node".